install_ffmpeg.sh

```diff
@@ -16,13 +16,15 @@ fi
 if [ ! -e "$HOME/x264/x264" ]; then
   git clone http://git.videolan.org/git/x264.git "$HOME/x264"
   cd "$HOME/x264"
+  # git master as of this writing
+  git checkout 7d0ff22e8c96de126be9d3de4952edd6d1b75a8c
   ./configure --prefix="$HOME/compiled" --enable-pic --enable-static
   make
   make install-lib-static
 fi
 
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
-  git clone https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
+  git clone -b n4.0 https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
```

Comment:
Just curious - is this branch definitely frozen or would it be better to checkout a specific commit similar to what is done for x264? (Same question for nasm as well)

Reply:
These are actually tags (rather than branches), so they should be frozen.  Unless the project decides to rewrite history... in which case all bets are off.